### DISCUSSION TITLE
Saisie d'un entier

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#233] [Annonces]
Est-ce qu'il est possible d'avoir un champ numérique qui accepte _seulement_ les entiers ?

**Comment by loima02** [comment]
Bonjour @ChrisThe01 ,

Le contrôle numérique entiers seulement est au four. Ça devrait être prêt ce pm lorsque les essais seront terminés.

Merci!

**Comment by anisite** [comment]
issue #234

**Comment by ChrisThe01** [comment]
Salut Mathieu,

J'ai trouvé la solution suivant :


  1.  Au lieu de :

            - name: nbrSalariesMoins28HeureSemaine
              type: number
              label:
                fr: Nombre des salariés permanents à temps partiel à l'emploi de l'organisme
              help:
                fr: Moins de 28 heures/semaine
              outerClasses: md
              validations:
                number:



  1.  J'ai pensé à ça :

            - name: tester
              type: number
              label:
                fr: Mon test
              outerClasses: md
              validations:
                number:
                matches: /^\d{1,7}$/

J'en penses-tu ?

Christian Thériault
Analyste en informatique
Division versement et systèmes de soutien
Service des systèmes partagés et compte-client
Direction des systèmes partagés, du travail et de l'assurance parentale
Direction générale des technologies de l'Information
Ministère du Travail, de l'Emploi et de la Solidarité sociale

425, Jacques-Parizeau, 3e étage
Québec (Québec) G1R 4Z1
Téléphone (local) : 581-628-8934 poste 80869
Téléphone (sans frais) : 1-888-628-8934 poste 80869
***@***.******@***.***>
www.mtess.gouv.qc.ca<http://www.mtess.gouv.qc.ca/>
[Je me protège et je sauve des vies]
Ce courriel peut contenir des renseignements confidentiels en vertu de la Loi sur l'accès aux documents des organismes publics et sur la protection des renseignements personnels. Ces renseignements ne peuvent être utilisés que par le destinataire mentionné dans l'en-tête. Si ce courriel vous est parvenu par erreur, nous vous informons que son usage est formellement interdit. Vous devez signaler cette erreur dans les meilleurs délais en communiquant avec l'expéditeur et détruire ce courriel. Nous vous remercions de votre collaboration.

De : Mathieu Loiselle ***@***.***>
Envoyé : 2 mai 2024 11:41
À : MTESSDev/FRW ***@***.***>
Cc : Thériault, Christian ***@***.***>; Mention ***@***.***>
Objet : [EXTERNE] Re: [MTESSDev/FRW] Saisie d'un entier (Discussion #233)

*ATTENTION : Ce courriel provient de l'extérieur de votre organisation.
Évitez de cliquer sur un hyperlien, d'ouvrir une pièce jointe ou de transmettre des informations personnelles si vous ne connaissez pas l'expéditeur du courriel. En cas de doute, communiquez verbalement avec lui.

Bonjour @ChrisThe01<https://github.com/ChrisThe01> ,

Le contrôle numérique entiers seulement est au four. Ça devrait être prêt ce pm lorsque les essais seront terminés.

Merci!

-
Reply to this email directly, view it on GitHub<https://github.com/MTESSDev/FRW/discussions/233#discussioncomment-9297157>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A62QKPLDEMSGDR2P76PG443ZAJNADAVCNFSM6AAAAABHDXWISKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TEOJXGE2TO>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

________________________________
Ce message est confidentiel et est à l'usage exclusif du destinataire nommé ci-dessus. Si vous recevez ce message par erreur, sachez qu'il est interdit de le diffuser, de le distribuer, d'en dévoiler le contenu ou de le reproduire. Veuillez informer immédiatement l'expéditeur par courrier électronique et détruire l'original de ce message ainsi que toute copie.
________________________________
